blocks/hero/hero.js

```diff
@@ -0,0 +1,37 @@
+import { buildBlock } from '../../scripts/lib-franklin.js';
+
+export default function decorate(block) {
+  const h1 = block.querySelector('h1');
+  const icon = block.querySelector('span');
+  const allPictures = block.querySelectorAll('.only-picture picture img');
+
+  const resolutions = ['1200px', '768px', '375px'];
+
+  icon.setAttribute('id', 'banner-icon');
+  const sectionTwo = document.createElement('div');
+  sectionTwo.append(icon);
+  sectionTwo.append(h1);
+  sectionTwo.setAttribute('class', 'logo-text-holder');
+
+  // create the <picture> element
+  const onePictureElement = document.createElement('picture');
+
+  const imgElement = document.createElement('img');
+  imgElement.setAttribute('src', 'path/to/image.jpg');
+  imgElement.setAttribute('id', 'imgcls');
+  imgElement.setAttribute('alt', 'Image description');
```

Comment:
@nimithshetty17 This looks to be a hard-coded alt-text description value. Let's remove this for now since we need to figure out a way to properly manage `alt` text value in Sharepoint doc/Franklin.

@amol-anand @helms-charity I noticed that some of the images that are embedded in the current docs do not have a column for managing alt text. This is a tech debt that we'll need to address in phase 3 to ensure that all of our images have a proper description. I'll create a ticket to capture this work.

Reply:
@proeung alt text is done in the Word document, so it is an authoring task.